### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by GiulioDioguardi**
Wat textuele verbeteringen

### CODE REVIEW

File: mysite/flask_app.py [hunk 1]
```diff
@@ -38,10 +37,12 @@ def map_page(questionhash):
     else:
         abort(404)
 
-    try:
-        return render_template('map.html', question=question, number=questionnumber)
-    except (IndexError, ValueError):
-        abort(404)
+    return render_template('map.html', question=question, number=questionnumber)
+
+@app.errorhandler(werkzeug.exceptions.NotFound)
```
Comment (mbunkers) [inline mysite/flask_app.py:42]: How does it get triggered? Is status 500 also a valid case just when some weird unexpected error occurs?

File: mysite/static/map.js [hunk 2]
```diff
@@ -45,11 +43,11 @@ function initMap() {
   });
 }
 
-function checkPosition(qTitle) {
+function checkPosition(qTitleHash) {
     if (google.maps.geometry.poly.containsLocation(marker.position, polygon)) {
         alert("Goed gedaan! Dit is het juiste antwoord.\nJe vraag wordt als goed gemarkeerd en je keert terug naar de vragenlijst");
```
Comment (mbunkers) [inline mysite/static/map.js:48]:
```suggestion
        alert("Goed gedaan! Dit is het juiste antwoord.\nJe vraag wordt als goed gemarkeerd en je keert terug naar de vragenlijst.");
```

File: mysite/flask_app.py (hunk 1 above)
Comment (GiulioDioguardi) [reply mysite/flask_app.py:42]: This is triggered when a page is not found. That exception is the typical 404 error.
I can also make another handler for the 500 range of errors.

File: mysite/static/map.js (hunk 2 above)
Comment (GiulioDioguardi) [reply mysite/static/map.js:48]: Yeah, some punctuation is always nice :)

File: mysite/flask_app.py (hunk 1 above)
Comment (mbunkers) [reply mysite/flask_app.py:42]: Ok, this is unknown to me as I don’t know how flask works.
That 500 status is up to you.

**Review by mbunkers — DISMISSED** [review]
LGTM.

File: mysite/flask_app.py (hunk 1 above)
Comment (GiulioDioguardi) [reply mysite/flask_app.py:42]: I've written something (or stolen from flask site) that, when I made a Jinja error of some kind that one can see what is going on.

Comment (GiulioDioguardi) [reply mysite/flask_app.py:42]: image

**Review by mbunkers — APPROVED** [review]

**Pull request merged**